just test command fails to run if lint fails

The original idea was a chain of command dependencies so one command can be run in merge automation. However, I expect test to run only go test and it's unhelpful to be blocked due to linting errors. Therefore, let's create a presubmit command that is dependent on both lint and test. Review other dependencies too. 

This is the current output when running `just test`:
```
... $ just test
cmd/revisio/main.go:32:14: Error return value of `fmt.Fprintf` is not checked (errcheck)
                fmt.Fprintf(stderr, "error: %v\n", err)
                           ^
internal/models/csvfile/csvfile.go:70:18: Error return value of `file.Close` is not checked (errcheck)
        defer file.Close()
                        ^
internal/models/csvfile/csvfile.go:92:18: Error return value of `file.Close` is not checked (errcheck)
        defer file.Close()
                        ^
internal/models/flashcard/flashcard.go:14:13: Error return value of `fmt.Fprintf` is not checked (errcheck)
        fmt.Fprintf(writer, "%s: %s\n", fc.key, fc.value)
                   ^
pkg/createcmd/createcmd.go:34:15: Error return value of `fmt.Fprintln` is not checked (errcheck)
                fmt.Fprintln(cfg.Stderr, rootcmd.AddFlagErr.Error())
                            ^
pkg/createcmd/createcmd.go:44:15: Error return value of `fmt.Fprintln` is not checked (errcheck)
                fmt.Fprintln(cfg.Stderr, rootcmd.AddFlagErr.Error())
                            ^
pkg/createcmd/createcmd.go:69:14: Error return value of `fmt.Fprintf` is not checked (errcheck)
                fmt.Fprintf(cfg.Stderr, "create %q OK\n", cfg.Subject)
                           ^
pkg/rootcmd/rootcmd.go:40:15: Error return value of `fmt.Fprintln` is not checked (errcheck)
                fmt.Fprintln(cfg.Stderr, AddFlagErr.Error())
                            ^
pkg/createcmd/createcmd.go:59:10: ST1005: error strings should not be capitalized (staticcheck)
                return errors.New("Missing required flag: -k, --key")
                       ^
pkg/createcmd/createcmd.go:61:10: ST1005: error strings should not be capitalized (staticcheck)
                return errors.New("Missing required flag: -v, --value")
                       ^
pkg/readcmd/readcmd.go:48:10: ST1005: error strings should not be capitalized (staticcheck)
                return errors.New("Missing required flag: -k, --key")
                       ^
pkg/rootcmd/rootcmd.go:15:2: ST1012: error var AddFlagErr should have name of the form ErrFoo (staticcheck)
        AddFlagErr = errors.New("unable to add flag")
        ^
12 issues:
* errcheck: 8
* staticcheck: 4
error: Recipe `lint` failed on line 14 with exit code 1
```

--- a/justfile
+++ b/justfile
@@ -13,8 +13,10 @@ vet: fmt
 lint: vet
   @golangci-lint run ./...
 
-test: lint
+test:
   @go test ./...
+
+presubmit: lint test
 
 alias b := build
 build: lint

